segments/vcs/vcs_git.spec

```
@@ -16,6 +16,12 @@ function setUp() {
   cd $FOLDER
   P9K_MODE=default
 
+  # Prevent the use of system or user specific gitconfig
+  OLD_HOME="$HOME"
+  HOME="$(dirname ${FOLDER})"
```

This should be equivalent to `dirname`
```suggestion
  HOME="${FOLDER:h}"
```